flat_pack.scad

```diff
@@ -0,0 +1,169 @@
+//openscad flat-pack joint library
```

Comment:
I'm not sure what tool you use to edit this file, but one small nit-picky detail that my tool (vim) shows is that some lines have stray whitespace at the end of them. It's not a big deal, but just a little thing that keeps your code cleaner and is generally good practice to look out for 😄

Reply:
I'm using vim too; What's your favorite way to visualize unneeded white space? I never really thought about this before. 
I found this useful: http://vim.wikia.com/wiki/Highlight_unwanted_spaces